src/cache.c

```diff
@@ -276,30 +272,6 @@ hsk_cache_key_set(hsk_cache_key_t *ck, const char *name, uint16_t type) {
     case 2:
       ref = !hsk_resource_is_ptr(name);
       break;
-    case 3:
```

Comment:
As discussed with @pinheadmz, this should have been removed even before this update. As root wont parse/recognize SRV/TLSA//SMIMEA and OPENPGPKEY records at all.